synthetic_data/distinct_generators/int_generator.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+from ..base_generator import BaseGenerator
+from numpy.random import Generator
+
+class IntGenerator(BaseGenerator):
+    def __init__(
+            self,
+            profile, #dataprofile
+            generator, #Random generator
+            ) -> None:
+        self.min = int(profile.report()["data_stats"][0]["statistics"].get("min", None))
+        self.max = int(profile.report()["data_stats"][0]["statistics"].get("max", None))
+        self.num_rows = profile.report()["global_stats"].get("column_count", None)
```

Comment:
I'm concerned this will be super brittle.

For now, let's limit the PR to just adding a function to do generation itself.

Reply:
also if its `column_count` ... wouldn't expect the `self.` assignment to be `num_rows` 🤔